app/root.tsx

```diff
@@ -22,7 +22,7 @@ export function Layout({ children }: { children: React.ReactNode }) {
       </head>
       <body className="bg-[#15181D] text-slate-50">
         <SiteNavigation />
-        {children}
+        <main className="min-h-[calc(100vh-304px)]">{children}</main>
```

Comment:
just worried, if there was a change in the footer height, did the code here also need to be updated? 
Why didn't you use flex like in the reference https://philipwalton.github.io/solved-by-flexbox/demos/sticky-footer/ instead of using a static min-height?

Reply:
got it